release/nightly_tests/nightly_tests.yaml

```diff
@@ -4,7 +4,7 @@
 
 # Test basic single node 10GB shuffle with a small number of partitions.
 # This doesn't require object spilling.
-- name: shuffle_10gb
+- name: shuffle_10gb_50_partitions
```

Comment:
Shall we leave the existing tests alone to avoid breaking the chart? I meant only change the names for the new tests (and that one 1tb test)